src/components/core/pivotTable/AggregationsMenu.tsx

```diff
@@ -175,8 +199,19 @@ export default class AggregationsMenu extends React.Component<IAggregationsMenuP
                     attributeIdentifier: attributeHeader.localIdentifier,
                 });
             const itemClassNames = this.getItemClassNames(totalType);
-            const renderSubmenu = showSubmenu && rowAttributeHeaders.length > 0;
-            const toggler = this.renderMenuItemContent(totalType, onClick, isSelected, renderSubmenu);
+            const isEnabled = totalType !== NATIVE_TOTAL || !afmFilters.find(AFM.isMeasureValueFilter);
+            const disabledItemTooltipText = isEnabled
```

Comment:
Please move this `const` in `renderMenuItemContent()`.

Reply:
I'm not sure what I was thinking about when I wrote this. It does not make any sense to do it like this 😐 Fixed.